packages/bpk-stylesheets/base.css

```diff
@@ -240,8 +240,7 @@ figure {
  * 2. Show the overflow in Edge and IE.
  */
 hr {
-  -webkit-box-sizing: content-box;
-          box-sizing: content-box;
+  box-sizing: content-box;
```

Comment:
Pretty sure this is fine. According to [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing), the vendor-prefixed version of `box-sizing` hasn't been needed since Safari 5.1, which was [released in 2011](https://en.wikipedia.org/wiki/Safari_version_history#Safari_5).